packages/runtime/src/errors/errors-list.ts

```diff
@@ -237,6 +237,12 @@ by an index that does not exist.`
     message: "set bit index %index% is beyond bytes length. [error-line: %line%]",
     title: 'set bit index error',
     description: `set bit index error`
+  },
+  SCRATCH_EXIST_ERROR: {
+    number: 1035,
+    message: "scratch space doesn't exist for index: %index%. [error-line: %line%]",
+    title: 'scratch space error',
```

Comment:
I think title can be more intuitive. This is related to the shared scratch space here, right?

Reply:
What do you propose ?

Reply:
Simple, `scratch space not found`/`scratch space does not exist`

Reply:
same as message, OK